.github/workflows/unit-test.yml

```diff
@@ -7,7 +7,7 @@ on:
         - master
 
 jobs:
-  build:
+  baseline-coverage:
```

Comment:
And maybe a better name as this job does more than `baseline-coverage`?

Reply:
done